modules/auxiliary/gather/jasmin_ransomware_dir_traversal.rb

```diff
@@ -0,0 +1,76 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
```

Comment:
For both of these modules - do they need to include the scanner library? Could this be removed and could `run_host` be changed to `run`?  Just curious.

Reply:
theyre in `gather` not `scanner` so you are correct!